geeksurvey/templates/base.html

```diff
@@ -63,7 +63,16 @@
       <!-- Footer-->
       <footer class="py-3 bg-dark">
         <div class="container px-5">
-          <p class="m-0 text-center text-white"> <a class="m-0 text-center text-white text-decoration-none" href="working.html"> Help </a></p>
+          <!-- If the current page is not in the same directory as the working.html page, navigate to it when the help link is used. -->
+          <!-- NOTE/TODO: The if and elif statements use the same reference the same relevant link to working.html. I tried combining the statements but that prevented the rest of the footer from appearing. Not sure why. -->
+          {% if "/accounts" in request.path %}
+            <p class="m-0 text-center text-white"> <a class="m-0 text-center text-white text-decoration-none" href="../../working.html"> Help </a></p>
```

Comment:
I think this should be fixed by using a proper url
```{% url 'working.html' %}```

and potentially a change to the working.html url in urls.py

Reply:
I was able to get it working using:
```{% url 'working' %}```
Also it doesn't seem to require a change to the working.html url in urls.py. I will commit the change soon!